cmd/osv-scanner/main_test.go

```diff
@@ -1298,7 +1299,8 @@ Filtered 2 vulnerabilities from output
 										"column": {
 											"start": 5,
 											"end": 6
-										}
+										},
+										"file_name": null
```

Comment:
If we add `omitempty` to the `FileName` property, will it that avoid sending it here a `nil`?